javascripts/tracking.js

```diff
@@ -68,6 +70,8 @@ piwikUsageTracking.createTrackersIfNeeded = function ()
 
     function anonymizeTitle(tracker)
     {
+        if (!tracker.isAnon) return;
```

Comment:
Can you add tests for this and the other methods eg in https://github.com/piwik/plugin-AnonymousPiwikUsageMeasurement/blob/master/javascripts/tracking.spec.js